contributing/releasing.md

```diff
@@ -33,6 +33,24 @@ For Googlers, also read [go/mdc-release-engineering](http://go/mdc-release-engin
 
 ## Before you start
 
+### First time setup
+
+Install [brew](https://brew.sh/), [sourcekitten](https://github.com/jpsim/SourceKitten), and [git-lfs](https://git-lfs.github.com/):
+
+    brew install sourcekitten
+    brew install git-lfs
```

Comment:
The instructions for git-lfs are already included below. Do you want to reorder sections a bit instead?